### PR TITLE
DBZ-6112 REPLICA IDENTITY setting

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java [hunk 1]
```diff
@@ -998,7 +998,7 @@ public byte[] getUnavailableValuePlaceholder() {
         return placeholder.getBytes();
     }
 
-    public ReplicaIdentityMapper getTableToReplicaIdentityMap() {
+    public ReplicaIdentityMapper replicaIdentityMapper() {
```
Comment (miguelbirdie) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java:1001]: @jpechane Happy to include this change in our branch 🙂

**Comment by miguelbirdie** [comment]
@jpechane I see that you are trying to remove `withReplicaIdentity`. Do you think that tests are failing because of that?

**Comment by jpechane** [comment]
@miguelbirdie I cahnged the code and hopefully the failures will be fixed. Still I have one more proposal for an imporvement. Right now you use table names. This should be switch to regular expression based mapping in the same way like table include list etc. work. The reason for it is that with regex you could easily set identify for all tables, per schema tables etc.

**Comment by miguelbirdie** [comment]
> @miguelbirdie I cahnged the code and hopefully the failures will be fixed. Still I have one more proposal for an imporvement. Right now you use table names. This should be switch to regular expression based mapping in the same way like table include list etc. work. The reason for it is that with regex you could easily set identify for all tables, per schema tables etc.

Ok, let me know when you fix these failures to rebase my branch with yours. @jpechane regarding regular expression, I think it's a good idea. Is there a function in the code to translate a regular expression to a list of tables?

**Comment by jpechane** [comment]
@miguelbirdie PLease take a look at `io.debezium.function.Predicates` and `io.debezium.util.Strings.setOfRegex(String)` as starting point.

**Comment by jpechane** [comment]
@miguelbirdie You can take it over from here.

**Pull request closed without merging**